network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/primary_heroguts.html

```diff
@@ -22,12 +30,27 @@
           {# Fallback Image #}
           {% image banner fill-1008x288 alt="" %}
         </picture>
+        {% endif %}
       {% else %}
         <img src="{% static "_images/banner-black-white-marble.jpg" %}" alt="">
       {% endif %}
     {% endwith %}
+
+    {% if homepage %}
```

Comment:
We're checking the page type in the template, which isn't bad or bad practice. But I'm wondering if this would be better split out for future hero sections (to keep it simple) or if checking the page type is going to be the easiest route for this fragment.

Reply:
@KalobTaulien mind creating a ticket and tag Pomax and myself for discussion? If this was a PR against `master` our visual regression tests (Percy) would kick off and flag issues for us but unfortunately, that's not the case here. 😬  I don't wanna change this file drastically in this PR because that could introduce unwanted changes to other pages without us knowing.

Reply:
@mmmavis Understood, that's a fair concern. New ticket here: #5039 5039